style.css

```diff
@@ -231,141 +231,80 @@ body {
   margin-left: 49%;
 }
 
-/* popup desktop */
+.title-1 {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  margin-left: -5%;
+}
+
+/* popup mobile */
+
 .main {
+  width: 375px;
+  height: 890px;
   position: fixed;
-  width: 2440px;
-  height: 1600px;
-  top: 900px;
-  background: #dfe0e2;
-  mix-blend-mode: normal;
-  margin-left: 1800px;
-  margin-top: -1000px;
-  z-index: 1;
+  top: 0;
+  background: #c1c7d0;
+  overflow-y: scroll !important;
 }
 
 .maincard {
-  grid-column: 8;
   display: flex;
   flex-direction: column;
   align-content: center;
   align-items: center;
-  border-radius: 8px;
-  margin-top: 204px;
-  margin-left: 960px;
-  width: 920px;
-  height: 1530px;
+  width: 343px;
   background: #fff;
-  border: 1px solid #ebecf0;
   box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
   z-index: 1;
+  height: 860px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 20px;
+  margin-bottom: 26px;
+  background-color: white;
+  border-radius: 16px;
 }
 
 .popup-image {
-  background-repeat: no-repeat;
-  width: 872px;
-  height: 616px;
-  left: 24px;
-  top: 78px;
-  border: 1px solid #dfe1e6;
-  border-radius: 8px;
+  width: 90%;
+  margin: 16px;
+  height: 230px;
+  display: block;
```

Comment:
Great work so far. You are almost there. 

- [ ] As stated by the previous reviewer, kindly ensure that the popup size fits the screen just as in the actual design. It appears bigger and difficult to view. 
- [ ] The popup menu is still not centered, it seems to be tilted to the left side. Kindly check and fix it. 
- [ ] The level of transparency applied to the popup makes it diffult to read the contents as shown below, kindly check and update it. 


### Your design 


![Screenshot from 2023-03-18 21-37-14](https://user-images.githubusercontent.com/65233860/226141394-97438f3b-f073-4079-bc46-4794b445a4a6.png)


### Actual design 

![Screenshot from 2023-03-18 21-27-40](https://user-images.githubusercontent.com/65233860/226141402-dc044263-fb53-4c7c-9ddc-fce39500eda0.png)